src/Neo.Extensions/UnsafeData.cs

```diff
@@ -18,14 +18,16 @@ public static class UnsafeData
         /// </summary>
         /// <param name="value">The length of the data.</param>
         /// <returns>The size of variable-length of the data.</returns>
-        public static int GetVarSize(int value)
```

Comment:
I think it's because we don't allow these big numbers for avoid DoS

Reply:
exactly, value represents the length of the data, int is already large enough to our usage.

Reply:
I think this needs to be symmetric to WriteVarInt/ReadVarInt, both of which support `0xFF` prefix. Also, the method can be used to evaluate encoded size of any integers in which case passing big values is OK. 

The only problem probably is that int/long can technically be negative.

Reply:
like @roman-khimov said it should inline with `WriteVarInt/ReadVarInt`.

Take this in to account.
```csharp
public class Test
{
    public ulong checksum { get; set; } // using Write/ReadVarInt
}
```

Reply:
`value` is always positive now,  but maybe it's better to add a negative check here?